lib/signaling/v2/peerconnection.js

```diff
@@ -314,8 +317,7 @@ PeerConnectionV2.prototype._handleDataChannelEvent = function _handleDataChannel
 PeerConnectionV2.prototype._handleGlare = function _handleGlare(offer) {
   var self = this;
   return Promise.resolve().then(function() {
-    var rollback = new self._RTCSessionDescription({ type: 'rollback' });
-    return self._setLocalDescription(rollback);
+    return self._setLocalDescription({ type: 'rollback' });
```

Comment:
This is probably fine if we rewrap in an RTCSessionDescription inside `_setLocalDescription`. The WebRTC 1.0 APIs are supposed to accept a plain JSON object in the future (e.g., RTCSessionDescriptionInit).

Reply:
@markandrus We do rewrap it. If you look at the JSDoc for `._setLocalDescription`, then it is expecting to accept a Dict.

Reply:
It is weird that it would accept _only_ the dict form though, and I guess that's because it mutates it?